docs/_docs/hivemq/policies.md

```diff
@@ -0,0 +1,290 @@
+---
+nav_order: 1
+redirect_from: /docs/hivemq/policies.html
+---
+
+# Policies
+
+***
+
+The policies command of the HiveMQ command line offers a set of commands to work with policies using a HiveMQ Data
+Validation API endpoint.
+
+```
+mqtt hivemq policies
+```
+
+***
+
+## Commands
+
+| Command | Explanation                         |
+|---------|-------------------------------------|
+| get     | See [Get Policy](#get-policy)       |
```

Comment:
Is the ordering of the commands on the page based on frequency of access? A normal lifecycle would feel like create, get, list, delete. but it's completely fine if get and delete are the usual commands used.

Reply:
I had ordering by simplest to most complex in mind, but lifecycle ordering would work too. The REST API documentation uses list, create, delete, get. I don't know what the frequency of access would be since I'm not sure how customers would actually use it. What order would you suggest?

Reply:
I leave it up to you, but I guess every approach is fine.